docs/developer-guide/working-with-plugins.md

```diff
@@ -18,6 +18,21 @@ module.exports = {
 };
 ```
 
+### Preprocessors in Plugins
+
```

Comment:
This approach limits plugins to just one set of pre/postprocessors. I don't think such a restriction makes sense.